Tests/AppTests/Helpers/ActorIsolated.swift

```diff
@@ -0,0 +1,113 @@
+// Copyright 2020-2023 Dave Verwer, Sven A. Schmidt, and other contributors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+// From https://github.com/pointfreeco/swift-composable-architecture/blob/ed3a380f09a81e72636d7b0699bf2dd2e6313780/Sources/ComposableArchitecture/Effects/ConcurrencySupport.swift#L297
```

Comment:
I'm not sure how to deal with license/attribution here. Everything in this file is from TCA's `ConcurrencySupport.swift` file in the `Effects` folder.

Would we add pointfree's license (MIT) instead of our header?

```
MIT License

Copyright (c) 2020 Point-Free, Inc.

Permission is hereby granted, free of charge, to any person obtaining a copy
of this software and associated documentation files (the "Software"), to deal
in the Software without restriction, including without limitation the rights
to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
copies of the Software, and to permit persons to whom the Software is
furnished to do so, subject to the following conditions:

The above copyright notice and this permission notice shall be included in all
copies or substantial portions of the Software.

THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
SOFTWARE.
```

Reply:
Great question. I'd say this is reasonable in a completely unmodified form since it's not our code.

It's probably not the kind of thing we'd modify, but I wonder if it's worth adding a note that nothing in this file should be modified in place, but via extensions (which we can keep in a separate file under our standard license), or contributions back to the original repository?

Reply:
Yes. Ideally, the file would be unmodified apart from our comment not to modify somewhere near the top. I see they don't include the license at the top of their files, so I'd suggest:

- Their license (MIT)
- A comment saying not to modify
- Their unmodified source

Reply:
It'll be modified, because we're only using one class out of a larger file.